NwPluginAPI/Core/Extensions/ItemExtensions.cs

```diff
@@ -0,0 +1,80 @@
+namespace PluginAPI.Core
+{
+	/// <summary>
+	/// Item-related extensions.
+	/// </summary>
+	public static class ItemExtensions
+	{
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is an ammo.
+		/// </summary>
+		/// <param name="item">The item to be checked.</param>
+		/// <returns>Returns whether the <see cref="T:ItemType" /> is an ammo or not.</returns>
+		public static bool IsAmmo(this ItemType item)
+		{
+			return item is ItemType.Ammo9x19 or ItemType.Ammo12gauge or ItemType.Ammo44cal or ItemType.Ammo556x45 or ItemType.Ammo762x39;
+		}
+
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is a weapon.
+		/// </summary>
+		/// <param name="type">The item to be checked.</param>
+		/// <param name="checkMicro">Indicates whether the MicroHID item should be taken into account or not.</param>
+		/// <param name="checkJailBird">Indicates whether the checkJailBird item should be taken into account or not.</param>
+		/// <returns>Returns whether the <see cref="T:ItemType" /> is a weapon or not.</returns>
+		public static bool IsWeapon(this ItemType type, bool checkMicro = true, bool checkJailBird = true)
+		{
+			if (type is ItemType.GunAK or ItemType.GunCOM15 or ItemType.GunCOM18 or ItemType.GunCom45 or ItemType.GunCrossvec or ItemType.GunLogicer or ItemType.GunRevolver or ItemType.GunShotgun or ItemType.GunE11SR or ItemType.GunFSP9)
```

Comment:
Consider using a switch instead if you have an if statement checking the same variable for a lot of values:

```cs
switch (type)
{
    case ItemType.GunAK:
    case ItemType.GunCOM15:
    case ItemType.GunCOM18:
    case ItemType.GunCom45:
    case ItemType.GunCrossvec:
    case ItemType.GunLogicer:
    case ItemType.GunRevolver:
    case ItemType.GunShotgun:
    case ItemType.GunE11SR:
    case ItemType.GunFSP9:
        return true;
}
```

It is a bit easier to read.

Reply:
also switch will not make better performance if you use 
type is things or otherthing or ..